### DISCUSSION TITLE
Egy bizonyalthoz több TaxpointDate

### DISCUSSION THREAD

**Discussion by Macskafarka** [nav-gov-hu/eVAT#86] [Kérdések és válaszok - Gépi interfész]
1. Jól gondolom, hogy a pénzforgalmi elszámolású számlákhoz mind számla kibocsátónál, mind a számla befogadónál akár több
TaxpointDate is tartozhat (ha a részkiegyenlítések több napon történnek)?
2. Ha az egyik havi bevalló, a másik éves bevalló, akkor ezek időben jó távol kerülnek bevallásba?
3. Mivel számlabefogadó oldalon a TaxpointDate csak a a levonási jog megnyílása, és nem muszáj ekkor ténylegesen levonásba helyezni, hanem lehet később is, ezért a NAV nem fog sem ERROR-t, sem WARNING-ot küldeni akkor, ha ugyanazon pénzforgalmi áfás bizonylatnál a számlabefogadó korábbi áfa időszakban érvényesíti a későbbi TaxpointDate-s részkiegyenlítést?  Példával: Havi bevalló számlabefogadó esetén a pénzforgalmi számla 1. részletét 01. hónapban fizetik ki, a 2. részletét a 03. hónapban. A 2. részlet bekerül a 03. havi áfa bevallásba a 03.havi TaxpointDate-tel, az 1. részlet viszont csak a 04.havi bevallásba kerül bele 01.havi TaxpointDate értékkel. Ez ma megfelel a törvényi kereteknek,
4. Van-e arra valamilyen elvárás, hogy a pénzforgalmi áfás bizonylat részkifizetésekor melyik tételsorra kell tenni a pénzügyi kiegyenlítést, vagy azt a számlakiállító és a számlabefogadó egymástól függetlenül maga dönti el? Pl.: 2 soros pénzforgalmi áfás számla 1. sora 100.000 Ft alap, és FAD, a 2. sora 100.000 Ft alap és 27% áfa. Ha erre 100.000 Ft részkifizetés történik, akkor a számlakiállító bevallhatja akként, hogy a FAD-os tételre érkezett a pénz, és egyelőre nem fizet áfát, viszont a számlabefogadó bevallhatja akként, hogy ő úgy gondolja, hogy a 27%-os tételsor egy részét tekinti kiegyenlítettnek, ezért levonásba helyez 27%-os áfa összeget.

**Comment by renced42** [comment]
Kedves @Macskafarka 

Kösz a kérdéseket a szakmai oldal készíti a válaszokat

**Comment by NTCA-HM** [comment]
Kedves @Macskafarka 

1.	Jól gondolom, hogy a pénzforgalmi elszámolású számlákhoz mind számla kibocsátónál, mind a számla befogadónál akár több TaxpointDate is tartozhat (ha a részkiegyenlítések több napon történnek)?

Igen.
A taxpointDate egy új fogalom az adókódok mellett, amely többalakú mert forrásonként más-más szabály szól arról, hogy ide mi értendő. A mező értékének meghatározásánál a következő elveket szükséges követni:
-	Áfafizetési kötelezettséget keletkeztető ügylethez tartozó bizonylatnál azt a dátumot kell feltüntetni, amikor az Áfa tv. szerint az adófizetési kötelezettség keletkezik. Ennek az adóelszámolási időszakba szükséges esnie.
-	A levonási jogot keletkeztető ügylethez tartozó bizonylatnál azt a dátumot kell feltüntetni, amikor az Áfa tv. szerint az adólevonási jogosultság megnyílik. Ennek az időpontja az adóelszámolási időszakba, vagy annál korábbi időszakra kell esnie, de nem korábbra, mint amelyet egyébként az Áfa tv. megenged elszámolni.
-	**Az egyes bizonylatok kapcsán szem előtt kell tartani, hogy egy bizonylat több időszaki adóelszámolásban is szerepelhet. Ilyen például a pénzforgalmi elszámolás, mivel ekkor egy bizonylathoz több taxpointDate érték is tartozik.**

2.	Ha az egyik havi bevalló, a másik éves bevalló, akkor ezek időben jó távol kerülnek bevallásba?

Igen, ez így igaz.

3.	Mivel számlabefogadó oldalon a TaxpointDate csak a a levonási jog megnyílása, és nem muszáj ekkor ténylegesen levonásba helyezni, hanem lehet később is, ezért a NAV nem fog sem ERROR-t, sem WARNING-ot küldeni akkor, ha ugyanazon pénzforgalmi áfás bizonylatnál a számlabefogadó korábbi áfa időszakban érvényesíti a későbbi TaxpointDate-s részkiegyenlítést? Példával: Havi bevalló számlabefogadó esetén a pénzforgalmi számla 1. részletét 01. hónapban fizetik ki, a 2. részletét a 03. hónapban. A 2. részlet bekerül a 03. havi áfa bevallásba a 03.havi TaxpointDate-tel, az 1. részlet viszont csak a 04.havi bevallásba kerül bele 01.havi TaxpointDate értékkel. Ez ma megfelel a törvényi kereteknek,

Igen.

4.	Van-e arra valamilyen elvárás, hogy a pénzforgalmi áfás bizonylat részkifizetésekor melyik tételsorra kell tenni a pénzügyi kiegyenlítést, vagy azt a számlakiállító és a számlabefogadó egymástól függetlenül maga dönti el? Pl.: 2 soros pénzforgalmi áfás számla 1. sora 100.000 Ft alap, és FAD, a 2. sora 100.000 Ft alap és 27% áfa. Ha erre 100.000 Ft részkifizetés történik, akkor a számlakiállító bevallhatja akként, hogy a FAD-os tételre érkezett a pénz, és egyelőre nem fizet áfát, viszont a számlabefogadó bevallhatja akként, hogy ő úgy gondolja, hogy a 27%-os tételsor egy részét tekinti kiegyenlítettnek, ezért levonásba helyez 27%-os áfa összeget.

A számlaadat-szolgáltatásban a „pénzforgalmi elszámolás”, mint kötelező számlaadattartalom, fejadatként szerepel (InvoiceData/invoiceMain/invoice/invoiceHead/invoiceDetail/ cashAccountingIndicator ), mely az Áfa tv. XIII./A fejezetében meghatározott különös szabályok szerinti adózás alkalmazása esetében tölthető. Ebből kifolyólag az ilyen számlákon fordított adózás alá eső tétel nem is szerepelhet (a fordított adózású számlát a kibocsátásának megfelelő adómegállapítási időszakban kell bevallani, míg a pénzforgalmi elszámolásnál az általa belföldön, a pénzforgalmi elszámolás időszakában teljesített összes termékértékesítése, szolgáltatásnyújtása után - ide nem értve a termék 10. § a) pontja és 89. §-a szerinti értékesítését, valamint azon termékértékesítéseket, szolgáltatásnyújtásokat, amelyekre az adóalany az Áfa törvény második részében meghatározott más különös rendelkezéseket alkalmazza - az általa fizetendő adót az adót is tartalmazó ellenérték jóváírásakor, kézhezvételekor állapítja meg; és vallja be.
